src/index.html

```diff
@@ -10,13 +10,103 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>Moyo</title>
     <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
-      href="./style.css"
+    />
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="index.html">
+          <img
+            src="https://github.com/JenyBerezhna/layout_moyo-header/blob/master/src/images/logo.png?raw=true"
+            alt="Moyo Logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a
+              href="#"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#"
+              class="nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav-item">
```

Comment:
Add the blank line between the links 
```suggestion
          </li>
          
          <li class="nav-item">
```